build_oc.tool

```diff
@@ -158,11 +158,20 @@ package() {
     cp "${arch}/Shell.efi" "${dstdir}/${arch}/EFI/OC/Tools/OpenShell.efi" || exit 1
 
     efiDrivers=(
+      "ArpDxe.efi"
```

Comment:
Instead of alphabetical order, shall we classify by authors (e.g. ours vs external)?

Reply:
I think this will complicate maintaining this in the future, so I think we should leave it as it is